lapce-ui/src/tab.rs

```diff
@@ -945,6 +946,25 @@ impl LapceTab {
                     LapceUICommand::VoltInstalled(volt) => {
                         let plugin = Arc::make_mut(&mut data.plugin);
                         plugin.installed.insert(volt.id(), volt.clone());
+
+                        // if there is a value inside the installing map, remove it from there as soon as it is installed.
+                        plugin.installing.remove(&volt.id());
+                    }
+                    LapceUICommand::VoltInstalling(volt, progress) => {
+                        let plugin = Arc::make_mut(&mut data.plugin);
+
+                        if plugin.installing.contains_key(&volt.id()) {
+                            let elem = plugin.installing.get_mut(&volt.id());
+                            elem.unwrap().set_progress(*progress);
```

Comment:
This can be written as
```suggestion
                        if let Some(elem) = plugin.installing.get_mut(&volt.id()) {
                            elem.set_progress(*progress);
```

I believe this is a bit cleaner.